lib/engine/step/g_1856/track.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
```

Comment:
what's the point of this file? it looks copy and pasted

Reply:
It is copy and pasted; the difference here is that the 1856 tracker is used instead of the base one. Is there a cleaner way of doing this? I am still trying to figure out how to effectively use the framework

Reply:
Instead of duplicating the whole file, you should extend the default Track class, and then override the specifics with what you need. It is possible that the "include Tracker" in the base needs to be moved until a place you can override.

Reply:
The contents of the class have been removed and it extends from the base Track step